src/bundle/Resources/public/js/scripts/admin.search.filters.js

```diff
@@ -60,7 +63,8 @@
         const isModifiedSelected = !!lastModifiedSelect.value;
         const isCreatedSelected = !!lastCreatedSelect.value;
         const isCreatorSelected = !!searchCreatorInput.value;
-        const isEnabled = isContentTypeSelected || isSectionSelected || isModifiedSelected || isCreatedSelected || isCreatorSelected;
+        const isSubtreeSelected = !!subtreeInput.value;
```

Comment:
I suppose this is a regular input, right? If so, then it should be: `!!subtreeInput.value.trim().length`. Thanks to this you'll avoid checking a value with spaces only.

Reply:
This is hidden input filled from UDW, so don't know if this make a lot of difference in that case.

Reply:
Well, when I see the `selected` or any similar word in the variable name, then I would expect a boolean value by default. Having an input, not checkbox or radio, in this case might be confusing.
Read it out loud. To select a subtree you have to type anything in the input and the variable name is about selection.

Reply:
Same logic is applied to creator or dates input as well as you don't have checkboxes there.